.github/ISSUE_TEMPLATE/issue_template_user_kudos.md

```diff
@@ -16,8 +16,9 @@ assignees: ''
 I thank the developers of the project. <br>
 Please add my information to the file [USERS.cff](https://github.com/OpenEnergyPlatform/oemetadata/blob/production/USERS.cff).
 
-family-names: 
-given-names: 
-alias: 
-affiliation: 
-orcid: 
+family-names: ""
+given-names: ""
+alias: ""
+affiliation: ""
+orcid: "https://orcid.org/"
+comment: "https://"
```

Comment:
not sure what the comment means here ? :)

Reply:
I added the comment to add a link to the source where the package has been used or implemented.
It now serves as a list of references and users.